CRM/Activity/DAO/Activity.php

```diff
@@ -234,6 +234,13 @@ static function &fields() {
           'dataPattern' => '',
           'export' => true,
         ) ,
+        'activity_free_text_search' => array(
```

Comment:
Hi @ravi-sharma what i was meaning is i think this added code and the getFreeTextFields() need to move into https://github.com/civicrm/civicrm-core/blob/master/CRM/Activity/BAO/Activity.php and then called from there, Unless you are actually planning on adding a column to the civicrm_activity table this is the wrong place. 

The DAO class is designed to be just about mimicking the database, these get regenerated whenever the schema changes. Most code should either go in the form layer or in the BAO class which is for the Business logic